modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/java/com/liferay/frontend/data/set/sample/web/internal/display/context/FDSSampleDisplayContext.java

```diff
@@ -35,13 +39,30 @@
 public class FDSSampleDisplayContext {
 
 	public FDSSampleDisplayContext(HttpServletRequest httpServletRequest) {
+		_httpServletRequest = httpServletRequest;
+
 		_fdsRequestHelper = new FDSRequestHelper(httpServletRequest);
 	}
 
 	public String getAPIURL() {
 		return "/o/c/fdssamples";
 	}
 
+	public List<DropdownItem> getBulkActionDropdownItems() {
+		return ListUtil.fromArray(
+			new FDSActionDropdownItem(
+				PortletURLBuilder.createActionURL(
+					_fdsRequestHelper.getLiferayPortletResponse()
+				).setActionName(
+					"/to_do_action"
+				).setCMD(
+					Constants.DELETE
+				).buildString(),
+				"trash", "delete", "delete",
+				LanguageUtil.get(_httpServletRequest, "delete"), "delete",
+				null));
+	}
```

Comment:
I am not sure about adding this as a placeholder. I can already see bug reports that our delete option is not working :slightly_smiling_face: 

What we should ideally do is to extend `propsTransformer`. We are already supporting customization of single actions, we will need to extend this to bulk actions as well. It would be used something like:
```js
onBulkActionItemClick({action, selectedItemsData}) {
	if (action.data.id === 'someID') {
		console.log(`You selected ${selectedItemsData.getAllTitlesSomehow()}!`);
	}
},
```
With this, there would be no need for `PortletURLBuilder`, URL can just be `#`. Sidenote - I don't remember why I set it [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/java/com/liferay/frontend/data/set/sample/web/internal/display/context/FDSSampleDisplayContext.java#L54) to `#` and not `null`. We should be able to set it to `null`. I guess this complicates things down the line with `LinkOrButton`.

We can do this as a seperate task, if you wish. If so, I think we should not add a placeholder in this PR, just have selection with no actions.

Reply:
> What we should ideally do is to extend propsTransformer.

Yes, I like this option, similar to what is done with each item action.
I was not sure about the need of implementing an action in the portlet. 
Thxs!

Reply:
Done & pushed